src/nuxeo-cloud/day-to-day/security.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Security
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 400
+toc: true
+---
+
+The Nuxeo Cloud environments are built using Cloud and application security best practices - including support for encryption of data at rest and in transit and threat detection monitoring using Threat Stack.
+
+A single tenant deployment model is applied to each customer’s production and pre-production deployments - meaning no resources are shared between customers.
```

Comment:
```suggestion
All Production and Pre-Production environment are deployed as single tenants so that no resources are shared between customers or environments.
```